nativeruntime/cpp/androidfw/CursorWindow.cpp

```diff
@@ -27,7 +27,17 @@
 #include <log/log.h>
 #include <stdlib.h>
 #include <string.h>
+// #include <sys/mman.h>
+#if !defined(_WIN32)
 #include <sys/mman.h>
+#else
+#ifndef PORT_READ
```

Comment:
these can be removed ,ashmem_set_prot_region is a no-op on host anyway, maybe we shouldn't even bother calling that function.

Reply:
Maybe we will use `ashmem_set_prot_region` when supporting CursrWindow's parceable. Current implementation uses temporarily file as backend of `ashemem`. We have discussed in-memory implementation for it at https://github.com/robolectric/robolectric/pull/6830.

Reply:
But I don't know when and where we should support CursorWindow's parceable for SQLite.